.circleci/config.yml

```diff
@@ -1040,6 +1040,7 @@ jobs:
       # See: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105562#c27
       CMAKE_OPTIONS: -DSANITIZE=address -DPEDANTIC=OFF
       CMAKE_BUILD_TYPE: Release
+      MAKEFLAGS: -j 2
```

Comment:
This needs a comment. Otherwise in the future someone will be wondering why it was specifically 2 :)